adr/ADR-32.md

```diff
@@ -132,9 +137,9 @@ Returns a JSON having the following structure:
      */
     version: string,
     /**
-     * Subject where the service can be invoked
+     * Root subject of the service
      */
-    subject: string
+    root_subject: string
```

Comment:
Would it be useful to have another field for a list of endpoints, which may be empty?

Reply:
Not sure I get it - `endpoints` is this field. Or you mean something else?

Reply:
another field
```
root_subject: string
endpoints: list of string
```

Reply:
Ah, sorry, I thought this comment was on config... Correct, I think having endpoints on INFO is a good idea.

Reply:
rootSubject won't be part of the config.